osu.Game/Overlays/Profile/Sections/BeatmapsSection.cs

```diff
@@ -22,6 +22,7 @@ public BeatmapsSection()
                 new PaginatedBeatmapContainer(BeatmapSetType.Ranked, User, UsersStrings.ShowExtraBeatmapsRankedTitle),
                 new PaginatedBeatmapContainer(BeatmapSetType.Loved, User, UsersStrings.ShowExtraBeatmapsLovedTitle),
                 new PaginatedBeatmapContainer(BeatmapSetType.Pending, User, UsersStrings.ShowExtraBeatmapsPendingTitle),
+                new PaginatedBeatmapContainer(BeatmapSetType.Guest, User, UsersStrings.ShowExtraBeatmapsGuestTitle),
```

Comment:
Should be moved to just below ranked as per https://github.com/ppy/osu-web/issues/8828